### DISCUSSION TITLE
Serving static files with resteasy-servlet-spring-boot-starter in springboot

### DISCUSSION THREAD

**Discussion by srpsingh** [resteasy/resteasy#4301] [General]
I'm using resteasy-servlet-spring-boot-starter 6.2.0.Final with springboot 3.2.2. I'm not sure how can we serve static files from /META-INF/resources/api. https://docs.jboss.org/resteasy/docs/6.2.9.Final/userguide/ suggests that we can use filter org.jboss.resteasy.plugins.server.servlet.FilterDispatcher but it doesn't seem to help as the servlet HttpServlet30Dispatcher just sends not-found response back. Looking for pointers to resolve this.

**Comment by liweinan** [comment]
@srpsingh I have moved this question here: https://github.com/resteasy/resteasy-spring-boot/discussions/340
